client/src/styles/GlobalStyles.tsx

```diff
@@ -149,13 +151,17 @@ const darkMode: SerializedStyles = css`
 `
 
 const GlobalStyles: React.FC = () => {
-  //Preload img
-  React.useMemo(() => {
+  const darkMode = React.useMemo(() => {
+    // Preload img
     new Image().src = moonSvg
     new Image().src = sunSvg
+    // Calc dark mode
+    const systemDarkMode = window?.matchMedia(PREFERS_COLOR_SCHEMA)?.matches
+    const localStorageIsDarkMode = localStorage.get(LOCAL_STORAGE_IS_DARK_MODE)
+    return localStorageIsDarkMode === null ? systemDarkMode : localStorageIsDarkMode
```

Comment:
I think nullish coalescing operator could work here

```suggestion
    return localStorageIsDarkMode ?? systemDarkMode
```